### ISSUE TITLE
Trolley missing from tracker 

### ISSUE THREAD

**Issue by ajhodges** [codeforgreenville/trolley-tracker#147]
Sitting at nose dive right now, there's a trolley right outside going South on main. The tracker is currently only showing the baseball route as being active and is only showing one trolley, down in the West end... We may have a hardware issue.

**Comment by ajhodges** [comment]
We have not seen trolley 5 since the 11th...

![screenshot_20160522-142253](https://cloud.githubusercontent.com/assets/856738/15455653/c311ff00-2028-11e6-8946-36063f415a20.png)

**Comment by ajhodges** [comment]
@tkenney-gvl do you have any ideas? Is the service still active for that nexus? Is it still plugged in the overhead bin?

**Comment by bikeoid** [comment]
I saw the note about a trolley being out of service, so I set the # of trolleys to 1 in the pager.   But I never realized when it returned to service.

So something might have become unplugged during service.

**Comment by tkenney-gvl** [comment]
Sorry for the delay. I just realized this morning that I had the trolley folder in my Outlook collapsed and I have an automatic rule to put everything from github for the trolley into there automatically. Had over 30 unreads there so was catching up. :) Anyway, I should be able to get over there this afternoon and check out both to make sure they're working again. Thanks for letting me know!

**Comment by tkenney-gvl** [comment]
So I just got to check on the Trolley that wasn't tracking. It's actually the only one in use today (their other driver called in sick or something so they only had that 1 running). It happened to be the one that wasn't tracking, so I checked on it and the Nexus had no battery left. I checked the power adapters and found that the USB to car adapter being used was pretty lose and when pushed up all the way would start to slide out again immediately. While holding it in, and having the trolley running it started to charge (altough it would take a bit to turn on). They're going to go ahead and try taping it in place so that we'll be able to see if it comes on for the routes like it should after that and if that works, they'll do the other one as a precaution, too. I'll keep an eye on it tonight (they'll be running till about 11pm) and we'll see what happens. I'll keep you guys informed as to what we find. Thanks!
